src/Sulu/Bundle/LocationBundle/Tests/Unit/Content/Types/LocationContentTypeTest.php

```diff
@@ -22,10 +22,10 @@ class LocationContentTypeTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->nodeRepository = $this->getMock('Sulu\Bundle\ContentBundle\Repository\NodeRepositoryInterface');
-        $this->phpcrNode = $this->getMock('PHPCR\NodeInterface');
-        $this->suluProperty = $this->getMock('Sulu\Component\Content\Compat\PropertyInterface');
-        $this->mapManager = $this->getMock('Sulu\Bundle\LocationBundle\Map\MapManager');
+        $this->nodeRepository = $this->createMock('Sulu\Bundle\ContentBundle\Repository\NodeRepositoryInterface');
```

Comment:
is this method also available in phpunit 4.8? if yes we should avoid the update and keep PHP 5.5 at least in this PR. all the other things we can discuss in the issue